test/docfx.Test/data/jschema/draft7/definitions.json

```diff
@@ -1,7 +1,7 @@
 [
     {
         "description": "valid definition",
-        "schema": {"$ref": "http://json-schema.org/draft-07/schema#"},
```

Comment:
don't change this, this means we don't support the `definitions` test suite, `ref` test suite already covers inline definitions